tensorboard/webapp/tbdev_upload/tbdev_upload_dialog_component.scss

```diff
@@ -52,3 +52,20 @@ code {
   font-size: 14px;
   line-height: 1.5;
 }
+
+.bottom-buttons {
+  align-items: center;
+  display: flex;
+  justify-content: flex-end;
+}
+
+.close-button {
+  color: map-get($tb-foreground, secondary);
```

Comment:
I'm guessing this is supposed to be `secondary-text` not `secondary`?

from `tensorboard/webapp/theme/_tb_theme.template.scss`

Reply:
Done. Good catch.